app/api/mealsController/meals.controller.js

```diff
@@ -0,0 +1,83 @@
+import mealsDb from "../db/meals.db";
+
+class MealsController {
+  //   get all meals
+  getAllMeals(req, res) {
+    res.status(200).send({
+      success: true,
+      message: "meals retrieved successfully",
+      meals: mealsDb
+    });
+  }
+
+  //   add a meal
+  addMeal(req, res) {
+    if (!req.body.name || !req.body.description || !req.body.price) {
```

Comment:
Entering an empty string with multiple spaces would still create a meal. You want to validate against that as well.